app/pages/theme/style.css

```diff
@@ -46,17 +41,26 @@ li {
   outline: 2px solid var(--ag-theme-secondary);
 }
 
-h1 {
+h1 a {
   font-weight: bold;
-  color: var(--ag-theme-primary) !important;
+  color: var(--ag-theme-primary);
   text-decoration: none;
 }
 
-h2,h3,h4 {
-  text-decoration: underline !important;
-  text-decoration-color: var(--ag-theme-primary) !important;
+h2 a,h3 a,h4 a {
+  color: inherit;
+  text-decoration: none;
 }
 
 h1:hover::after, h2:hover::after, h3:hover::after, h4:hover::after {
   content: " 🔗";
+  text-decoration: none !important;
 }
+
+.depth1 {
```

Comment:
Would it be possible to change these classes to something like `.agregore-depth1` so that it'll be less likely to conflict with custom CSS styles?

Reply:
Sounds good.
Will do.